content/body/marquee.php

```diff
@@ -1,5 +1,6 @@
 <p>
-    Marquees are meant for content that scrolls or updates consistently, like a stock ticker or a news feed. Although the have a default <code>aria-live</code> value of <code>off</code>, you can use <code>aria-live="polite"</code> to let users hear the information within a marquee in almost real-time.  Do this carefully, the last thing you would want is have a screen reader update too much in a way that would make the rest of your application unusable to screen reader users due to too much screen reader noise.
+    Marquees are meant for content that scrolls or updates consistently, like a stock ticker or a news feed. 
+    Although they have a default <code>aria-live</code> value of <code>off</code>, you can use <code>aria-live="polite"</code> to let users hear the information within a marquee in almost real-time.  Do this carefully. It would be a bad user experience to have a screen reader update too often. The rest of your application will become unusable to screen reader users due to too much screen reader noise caused by too many alerts.
```

Comment:
I feel that the last 3 sentences should be tied together a bit better. What about the following?

```suggestion
    Although they have a default <code>aria-live</code> value of <code>off</code>, you can use <code>aria-live="polite"</code> to let users hear the information within a marquee in almost real-time.  Do this carefully since it would be a bad user experience to have a screen reader update too often. The rest of your application will become unusable to screen reader users due to too much screen reader noise when there are too many alerts.
```